client-v2/src/fixtures/initialState.ts

```diff
@@ -296,10 +296,10 @@ const shared = {
         type: 'fixed/small/slow-IV'
       }
     },
-    pagination: null,
-    lastError: null,
-    error: null,
-    lastFetch: null,
+    pagination: undefined,
```

Comment:
We don't use null types much elsewhere in the project, so I updated them here for consistency's sake -- sorry for the noise!